packages/terra-date-input/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 ## 1.38.0 - (June 5, 2023)
 
 * Changed
-  * Updated `uuid` to `v8` for consistency with other components.
+  * Updated `uuid` to `v8.2.0` for consistency with other components.
 
 ## 1.37.0 - (May 25, 2023)
```

Comment:
This has already been released so we'll need a separate entry under `unreleased`

Reply:
Good catch. Fixed in [this commit](https://github.com/cerner/terra-framework/pull/1672/commits/3e5e2003f859b32d6ee206d2a8f84f78ea10a3ab)